docs/en/datasets/det.md

```diff
@@ -212,3 +218,29 @@ rm dataset.zip && rm -rf dataset
 ```bash
 python tools/data/textdet/funsd_converter.py PATH/TO/funsd --nproc 4
 ```
+
+### SROIE
+
+- Step1: Download `0325updated.task1train(626p).zip`, `task1&2_test(361p)-20220304T064017Z-001`, and `text.task1&2-test（361p).zip` from [homepage](https://rrc.cvc.uab.es/?ch=13&com=downloads) to `sroie/`
+- Step2:
+
+```bash
+mkdir sroie && cd sroie
+mkdir imgs && mkdir annotations && mkdir imgs/training
+
+unzip -q 0325updated.task1train\(626p\).zip && unzip -q task1\&2_test\(361p\)-20220304T064017Z-001.zip && unzip -q text.task1\&2-test（361p\).zip
```

Comment:
```suggestion
unzip -q 0325updated.task1train\(626p\).zip && unzip -q task1\&2_test\(361p\).zip && unzip -q text.task1\&2-test（361p\).zip
```
The timestamp is created by GDrive! Might need to *note* it down.

Reply:
Forgot to delete it.